src/Microsoft.AspNet.Mvc.Core/OptionDescriptors/ModelBinderDescriptorExtensions.cs

```diff
@@ -80,5 +80,24 @@ public static class ModelBinderDescriptorExtensions
             descriptors.Insert(index, descriptor);
             return descriptor;
         }
+
+
+        /// <summary>
+        /// Removes all instances of <typeparamref name="TInstance"/> from a list of ModelBinderDescriptors.
+        /// </summary>
+        /// <typeparam name="TInstance">A type that represents an <see cref="IModelBinder"/>.</typeparam>
+        /// <param name="descriptors">A list of ModelBinderDescriptors.</param>
```

Comment:
Line 89 is consistent with the xml comment for the descriptors parameter in the other methods in `ModelBinderDescriptorExtensions`.  And also with methods in all the other Xxx`DescriptorExtensions`.  Probably best not to change it just here. :grimacing: